syncing/src/main/kotlin/maru/syncing/ChainTracker.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Consensys Software Inc.
+ *
+ * This file is dual-licensed under either the MIT license or Apache License 2.0.
+ * See the LICENSE-MIT and LICENSE-APACHE files in the repository root for details.
+ *
+ * SPDX-License-Identifier: MIT OR Apache-2.0
+ */
+package maru.syncing
+import maru.core.SealedBeaconBlock
+
+interface ChainTracker {
+  fun onChainHeadUpdated(chainHeadUpdatedHandler: ChainHeadUpdatedHandler)
+}
+
+interface ChainHeadUpdatedHandler {
+  fun onChainHeadUpdated(headSealedBeaconBlock: SealedBeaconBlock)
```

Comment:
Since we're going to rely on status messages for determining the chain head, I don't think `SealedBeaconBlock` should be here. I think it has to be `BeaconSyncTarget`